tutorials/nservicebus-101/index.md

```diff
@@ -0,0 +1,34 @@
+---
+title: "NServiceBus 101: Messaging Basics"
+suppressRelated: true
+---
+
+In this course, you will learn how to get working quickly with NServiceBus.
+
+
+## Objectives
+
+In this course we'll build a messaging system using a retail e-commerce domain.
+
+By the end of the course, you will have learned:
+
+ * How to set up your development environment
+ * What a messaging endpoint is and how to create one
+ * How to define messages and message handlers
+ * How to send messages between endpoints and control where those messages go
+ * How (and why) to publish events to multiple subscribers
+ * How to deal with exceptions and take advantage of automatic retries
+
+
+## Table of Contents
+
+ 1. [Lesson 1: Hello world](lesson-1/)
+ 1. [Lesson 2: Sending a command](lesson-2/)
```

Comment:
Lesson 4 is "Publishing events" so I guess this one should be "Sending commands"